Slim/Player/Playlist.pm

```diff
@@ -52,7 +52,12 @@ sub shuffleType {
 	return 'none';
 }
 
-sub song {
+sub song { 
+	$log->warn("use of Slim::Player::Playlist::song is deprecated, use track instead");
```

Comment:
You could add a `$log->is_info && logBacktrace('deprecated!')` here. We might want to know what was calling us the wrong way.

Reply:
Done